### DISCUSSION TITLE
Mutating the English JobFunnel codebase to Dutch version

### DISCUSSION THREAD

**Discussion by ghost** [PaulMcInnis/JobFunnel#157] [General]
I would like to receive an unambiguous, yet detailed explanation via email/github as soon as possible so that I understand how and what code i need to refactor from the **UK_ENGLISH** code-base to the **NL_DUTCH** code-base, 
so that JobFunnel is immediately available for Dutch Indeed. Because i need to find a job asap and the translated script will be a game changer and a nice portfolio for me. 

Thanks to your feedback,

I look forward to your response,

Best,
Rick

**Comment by PaulMcInnis** [comment]
Please review the french implementation, and do a similar thing for dutch version :+1:  

I cannot give you a detailed guide, sorry.
